src/components/content/order/serviceConfiguration/CurrentServiceConfiguration.tsx

```diff
@@ -0,0 +1,112 @@
+    userOrderableServiceVo: UserOrderableServiceVo | undefined;
+}): React.JSX.Element => {
+    const getDescriptionForKey = (key: string): string | undefined => {
+        const param = userOrderableServiceVo?.configurationParameters?.find((param) => param.name === key);
+        return param?.description ?? key;
+    };
+
+    return (
+        <div>
+            {serviceConfigurationDetails && userOrderableServiceVo ? (
+                <Form key='currentConfig' disabled={true}>
+                    <Row className={serviceOrderStyles.orderFormSelectionFirstInGroup}>
+                        <Col span={6} className={serviceOrderStyles.orderFormLabel}>
+                            <Form.Item
+                                label={
+                                    <p
+                                        className={`${serviceOrderStyles.orderFormSelectionStyle} ${serviceOrderStyles.orderFormItemName}`}
+                                    >
+                                        {'ServiceId'}
+                                    </p>
+                                }
+                                labelCol={{ style: { textAlign: 'left' } }}
+                            ></Form.Item>
+                        </Col>
+                        <Col span={18}>
+                            <Input name={'ServiceId'} defaultValue={serviceConfigurationDetails.serviceId} />
+                        </Col>
+                    </Row>
+                    <Row className={serviceOrderStyles.orderFormSelectionFirstInGroup}>
+                        <Col span={6} className={serviceOrderStyles.orderFormLabel}>
+                            <Form.Item
+                                label={
+                                    <p
+                                        className={`${serviceOrderStyles.orderFormSelectionStyle} ${serviceOrderStyles.orderFormItemName}`}
+                                    >
+                                        {'CreatedTime'}
+                                    </p>
+                                }
+                                labelCol={{ style: { textAlign: 'left' } }}
+                            ></Form.Item>
+                        </Col>
+                        <Col span={18}>
+                            <Input name={'CreatedTime'} defaultValue={serviceConfigurationDetails.createdTime} />
+                        </Col>
+                    </Row>
+                    <Row className={serviceOrderStyles.orderFormSelectionFirstInGroup}>
+                        <Col span={6} className={serviceOrderStyles.orderFormLabel}>
+                            <Form.Item
+                                label={
+                                    <p
+                                        className={`${serviceOrderStyles.orderFormSelectionStyle} ${serviceOrderStyles.orderFormItemName}`}
+                                    >
+                                        {'UpdatedTime'}
```

Comment:
we can just use this time step. but rename the field to 'Config Last Updated At'